README.md

```diff
@@ -4,24 +4,26 @@
 
 ## Buildkite Elastic CI Stack for AWS
 
-[Buildkite](https://buildkite.com/) is a platform for running fast, secure, and scalable continuous integration pipelines on your own infrastructure.
+[Buildkite](https://buildkite.com/) provides a platform for running fast, secure, and scalable continuous integration pipelines on your own infrastructure.
```

Comment:
Good points in relation to [your earlier comment](https://github.com/buildkite/elastic-ci-stack-for-aws/pull/1479#discussion_r2109612280), and in relation to [your next one](https://github.com/buildkite/elastic-ci-stack-for-aws/pull/1479#discussion_r2112395151), I decided to change the word 'is' to 'provides', because Buildkite is a company that produces software products, one of which is the [Buildkite platform](https://buildkite.com/docs/platform). Hence, saying Buildkite [the company] _provides_ a platform is more accurate.